README.md

```diff
@@ -58,3 +60,5 @@ wrote ./jenkins/templates/jenkins-agent-svc.yaml
 wrote ./jenkins/templates/jenkins-master-svc.yaml
 wrote ./jenkins/templates/jenkins-master-deployment.yaml
 ```
+
+If the chart repository needs authentication, provide credentials via `--username` and `--password`.
```

Comment:
As we are using `viper.AutomaticEnv()` environment variables should also work. Could you verify this manuall and also document it here in the README?

Reply:
Currently, we do not bind flags and only the command line parameters will work.